embeddings/pipeline/hf_preprocessing_pipeline.py

```diff
@@ -27,16 +26,17 @@ def __init__(
         dataset_name: str,
         persist_path: str,
         sample_missing_splits: Optional[Tuple[Optional[float], Optional[float]]] = None,
+        downsample_splits: Optional[
+            Tuple[Optional[float], Optional[float], Optional[float]]
```

Comment:
Consider adding new file with just common types definitions to import as now you repeat quite long definitions in couple of places like here and in embeddings/pipeline/preprocessing_pipeline.py etc

Reply:
Done, added it to `__init__` file in `pipelines` module

Reply:
How about types.py?

Reply:
I would add it to the refactor section and move all type aliases. Added to #120 ideas.